script/sysroots.json

```diff
@@ -1,36 +1,36 @@
 {
     "bullseye_amd64": {
-        "Sha1Sum": "202e5738f4fad834a43ad9978efc53ff710ee979",
+        "Sha1Sum": "8068722ee05e9c672f9c29f32fd08acc0539d0e8",
         "SysrootDir": "debian_bullseye_amd64-sysroot",
         "Tarball": "debian_bullseye_amd64_sysroot.tar.xz"
     },
     "bullseye_arm": {
-        "Sha1Sum": "683d994e5643dff423643eac59c1f62606d14f43",
+        "Sha1Sum": "835cb5f1f40618de7cbd411e83128adc5697e426",
         "SysrootDir": "debian_bullseye_arm-sysroot",
         "Tarball": "debian_bullseye_arm_sysroot.tar.xz"
     },
     "bullseye_arm64": {
-        "Sha1Sum": "5a56c1ef714154ea5003bcafb16f21b0f8dde023",
+        "Sha1Sum": "14c1ca201a217daff6a90f5369bbf2b42cde8d77",
         "SysrootDir": "debian_bullseye_arm64-sysroot",
-        "Tarball": "debian_sid_arm64_sysroot.tar.xz"
+        "Tarball": "debian_bullseye_arm64_sysroot.tar.xz"
```

Comment:
This would bring back the bug addressed in https://github.com/electron/electron/pull/34491, there is a small change required in upstream sysroot script for us to use bullseye https://bugs.chromium.org/p/chromium/issues/detail?id=1309965#c5 and I just found that upstream is willing to take a CL for it.

Reply:
Can we raise an issue pre-emptively and mark it as a release blocking?

Reply:
https://github.com/electron/electron/issues/35877